pkg/core/blockchain.go

```diff
@@ -1290,6 +1320,9 @@ func (bc *Blockchain) verifyAndPoolTx(t *transaction.Transaction, pool *mempool.
 		return fmt.Errorf("%w: %v", ErrPolicy, err)
 	}
 	size := t.Size()
+	if isPartialTx { // then it's a fallback transaction from NotaryRequest with missing Notary witness
+		size += 66 // InvocationScript: 2 bytes for PUSHDATA1 + 64 bytes for signature; VerificationScript: 0 bytes (empty script)
```

Comment:
What if we're to require invocation script to be "almost" correct? In that it will `PUSHDATA1(0x000...)`, which will make tx size correct and allow to run verification script as usual (even though it will fail).

Reply:
Done, and also added several checks to NotaryRequest.DecodeBinary. We have to check the length of both invocation and verification scripts for dummy witness, and also have to check the PUSHDATA1 prefix in order to match the desired verification price exactly.